{{cookiecutter.project_shortname}}/scripts/bootstrap

```diff
@@ -5,7 +5,7 @@ set -e
 pip install -r requirements.txt
 pip install -e .
 
-npm update && npm install --silent -g node-sass@3.8.0 clean-css@3.4.19 uglify-js@2.7.3 requirejs@2.2.0
+npm update && npm install --silent -g node-sass clean-css@3.4.19 uglify-js@2.7.3 requirejs@2.2.0
```

Comment:
I guess since tests pass now, we can pin the latest version which is installed by default.